tests/airdrop_index_v2_schema.json

```diff
@@ -19,19 +19,35 @@
                             "type": "string",
                             "pattern": "^[a-fA-F0-9]{64}$"
                         },
-                        "api_url": {"type": "string", "format": "uri"},
-                        "amount_path": {"type": "string"},
+                        "api_url": {
+                            "type": "string",
+                            "format": "uri"
+                        },
+                        "amount_path": {
+                            "type": "string"
+                        },
                         "asset_identifier": {
                             "type": "string"
                         },
                         "url": {
                             "type": "string",
                             "format": "uri"
                         },
-                        "name": {"type": "string"},
-                        "icon": {"type": "string"},
-                        "icon_path": {"type": "string"},
-                        "cutoff_time": {"type": "integer"},
+                        "name": {
```

Comment:
Why this change?

Reply:
is due to auto-formatting it, earlier it was inconsistent

Reply:
also added the `has_decoder` field in the schema